tests/expected/functions.out

```diff
@@ -265,7 +265,7 @@ SELECT date_trunc('day', c at time zone 'UTC') as d1 FROM t2 GROUP BY d1 ORDER B
  2019-01-02 00:00:00
 (2 rows)
 
-EXPLAIN (VERBOSE, COSTS OFF) SELECT extract('day' from c at time zone 'UTC') as d1 FROM t1 GROUP BY d1 ORDER BY d1;
+EXPLAIN (VERBOSE, COSTS OFF) SELECT date_part('day'::text, timezone('UTC'::text, c)) as d1 FROM t1 GROUP BY d1 ORDER BY d1;
```

Comment:
Previous versions of the Postgres `extract` function used the underlying function `date_part`, but in Postgresql 14 it was changed because of loss-of-precision issues in some usages. Now `extract` function has its implementation.

Reply:
```
 EXPLAIN (VERBOSE, COSTS OFF) SELECT extract('day' from c at time zone 'UTC') as d1 FROM t2 GROUP BY d1 ORDER BY d1;
                                                                                 QUERY PLAN                                                                                
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  Foreign Scan
-   Output: (date_part('day'::text, timezone('UTC'::text, c)))
+   Output: (EXTRACT(day FROM (c AT TIME ZONE 'UTC'::text)))
    Relations: Aggregate on (t2)
-   Remote SQL: SELECT toDayOfMonth(toTimeZone(c, 'UTC')) FROM regression.t1 GROUP BY (toDayOfMonth(toTimeZone(c, 'UTC'))) ORDER BY toDayOfMonth(toTimeZone(c, 'UTC')) ASC
+   Remote SQL: SELECT extract('day', toTimeZone(c, 'UTC')) FROM regression.t1 GROUP BY (extract('day', toTimeZone(c, 'UTC'))) ORDER BY extract('day', toTimeZone(c, 'UTC')) ASC
 (4 rows)
```